src/main/java/com/kryptnostic/conductor/rpc/odata/Table.java

```diff
@@ -137,13 +138,44 @@ static CassandraTableBuilder createTableDefinition( Table table ) {
                         .ifNotExists()
                         .partitionKey( NAME )
                         .columns( SECURABLE_OBJECTID );
+
+            // TODO: ENTITY_ID( DataType.uuid() ) or ENTITYID( DataType.text() )
+            case AUDIT_LOG:
+                return new CassandraTableBuilder( AUDIT_LOG )
+                        .ifNotExists()
+                        .partitionKey(
+                                CommonColumns.ACL_KEYS,
+                                CommonColumns.EVENT_TYPE
+                        )
+                        .clusteringColumns(
+                                CommonColumns.PRINCIPAL_TYPE,
+                                CommonColumns.PRINCIPAL_ID,
+                                CommonColumns.TIME_UUID
+                        )
+                        .columns(
+                                CommonColumns.ENTITY_ID,
```

Comment:
I'd be ok with `VERTEX_ID`, but the table contains 2 columns that store vertex IDs, so I wanted something more specific. What about `ENTITY_DATA_ID` + `AUDIT_ID`, or `ENTITY_VERTEX_ID` + `AUDIT_VERTEX_ID`?